assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/edit.tsx

```diff
@@ -24,8 +26,8 @@ export const Edit = ( {
 	setAttributes: ( attributes: Record< string, unknown > ) => void;
 } ): JSX.Element => {
 	const blockProps = useBlockProps();
-	const currentText = text || termsCheckboxDefaultText;
-
+	const currentText =
+		text || checkbox ? termsCheckboxDefaultText : termsConsentDefaultText;
```

Comment:
I think it's caused by this line, but if I modify the text and then reload the page, changes are no longer there:

https://user-images.githubusercontent.com/3616980/135092208-c57845e2-de58-4eab-9d10-30f3821501d8.mp4

Reply:
Fixed this, the value persist but isn't shown because I didn't wrap `checkbox ? termsCheckboxDefaultText : termsConsentDefaultText` in `( )`

Reply:
Confirming this is fixed now, thanks!